tests/prague/eip6110_deposits/test_modified_contract.py

```diff
@@ -0,0 +1,223 @@
+"""Test variants of the deposit contract which adheres the log-style as described in EIP-6110."""
+
+import pytest
+
+from ethereum_test_exceptions.exceptions import TransactionException
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Block,
+    BlockchainTestFiller,
+    Header,
+    Requests,
+    Transaction,
+)
+from ethereum_test_tools import Macros as Om
+from ethereum_test_tools import Opcodes as Op
+
+from .helpers import DepositRequest, create_deposit_log_bytes
+from .spec import Spec, ref_spec_6110
+
+pytestmark = pytest.mark.valid_from("Prague")
```

Comment:
```suggestion
pytestmark = [
    pytest.mark.valid_from("Prague"),
    pytest.mark.execute(pytest.mark.skip(reason="modifies pre-alloc")),
]
```
This is a nice addition to have to avoid trying to execute these tests in live networks, since it requires a modified contract which we cannot pre-set there.